src/daemon/ca/certauth.rs

```diff
@@ -1018,19 +1003,20 @@ impl CertAuth {
 
             let mut issued_certs = vec![];
             for key in certified_keys {
-                if let Some(issued) = rc.issued(&key) {
+                if let Some(issued) = rc.delegated(&key) {
```

Comment:
There is some history behind using 'delegated' in Krill vs issued in the RFC 6492 Resource Class List Response context. It was intended to avoid confusion. But.. I agree it's best to use the term 'issued' wherever we can. So, now changed.